app/views/complaints/show.html.erb

```diff
@@ -7,17 +7,35 @@
   <div class="grid-row grid-gap">
     <div class="grid-col-4">
       <div class="summary-section">
-        <h2>Summary</h2>
-        <p>
-          <strong><%= @complaint.grantee %>:</strong>
-          <%= @complaint.summary %>
-        </p>
+        <div class="ct-section-divider">
+          <h2>Summary</h2>
+          <p>
+            <strong><%= @complaint.grantee %>:</strong>
+            <%= @complaint.summary %>
+          </p>
+        </div>
+        <% unless @tta_reports.empty? %>
```

Comment:
[suggestion] I think that we could use `<% if @tta_reports.present? %>` here instead, which to me seems more intuitive than "unless empty"